### PR TITLE
Correct typo in protocol-name description

### PR DESCRIPTION
**Pull request by qm2k**
Description stated that this leaf should match identifier of a protocol, but it should actually match name of a protocol.

Fixes: #1184

### Change Scope

Comment-only change preventing misinterpretation of the leaf meaning.

### CODE REVIEW

File: release/models/network-instance/openconfig-network-instance-policy.yang [hunk 1]
```diff
@@ -74,7 +74,7 @@ revision "2018-11-21" {
           on in the local network instance. The string
           must match one of /network-instances/
           network-instance/protocols/
-          protocol/identifier in the local network
+          protocol/name in the local network
```
Comment (earies) [inline release/models/network-instance/openconfig-network-instance-policy.yang:77]: The typo correction is correct however it's worth pointing out that these constructs are heavily relying on description statements to describe the intent vs. formulating proper model restrictions to enforce existence of these list keys within a local NI context.

PR as-is LGTM but this modeling should be reworked as a subsequent issue as I'm noticing now.

It requires implementations to have to build special logic outside of the schema that can provide inconsistencies.  This identityref allows for _ANY_ protocol type to be referenced legally and any unbound string for the name.  Maybe the grouping should not be so loose especially when there is only 1 current usage

**Review by dplore — COMMENTED** [review]
You will need to increment the 'patch' version:

ie:


`ext:openconfig-version "0.1.3";`

**Comment by qm2k** [comment]
> You will need to increment the 'patch' version:
> 
> ie:
> 
> `ext:openconfig-version "0.1.3";`

Thank you, done.

File: release/models/network-instance/openconfig-network-instance-policy.yang (hunk 1 above)
Comment (qm2k) [reply release/models/network-instance/openconfig-network-instance-policy.yang:84]: Anything I can do to help it? I am implementer, and I needed this ambiguous point resolved. I would of course always welcome more clarity on the schema level, but for now correct description is better than nothing. I would prefer to keep an issue of matching against technically unbound space of potential protocol names separate.

Comment (rgwilton) [reply release/models/network-instance/openconfig-network-instance-policy.yang:84]: LGTM

**Review by dplore — APPROVED** [review]

**Pull request merged**